index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Component-Library</title>
+    <link href="style.css" rel="stylesheet" />
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" rel="stylesheet"/>
+</head>
+<body>
+    
+    <nav class="navigation container">
+        <div class="nav-brand"></div>
+        <ul class="list-non-bullet nav-pills">
+            <li class="list-item-inline-components">
+                <a class="link1" href="./components.html">Components</a>
+            </li>
+            <li class="list-item-inline-github">
+                <a class="link2" href="https://github.com/mansiagarwal451/My-Component-library">Github</a>
+            </li>
+           
+        </ul>
+    </nav>
+    <hr>
+
+    <div class="welcome"><h3>Welcome to my component library!</h3></div>
+    <img src="html_css_beginners_banner.png" alt = "/">
+    </div>
```

Comment:
if you want the image to be centered,
you just have to include the ```img``` element inside the div

Reply:
Thanks Roshan for reviewing my code! I have made the suggested changes and updated my component library.